docs/coding_standards.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: Adapted Strength Coding Standards
+Version: 1.0.0
+---
+
+## Document Disclaimer and Purpose
+This document 
+
+## Document Standards
+
+### Document Version
+This docuemnt SHALL be versioned as follows: Major. Minor. Patch. These are defined as follows.
+- A change in the major version SHALL signify major changes to this docuement, such as adding or removing large sections of this document. 
+- A change in the minor version SHALL signfiy minor changes to this document, such as reworking a section that does not effect the wider document.
+- A change in the patch version SHALL signify changes that do not effect the understanding of this doucment, such as rewording sections for clarity, or fixing spelling or grammar
+
+### Document Standards
+This document SHALL follow RFC standards [RFC 8174](https://www.rfc-editor.org/rfc/rfc8174) and [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119).
+These standards establish the following:
+ - MUST, REQUIRED, and SHALL are to be used to establish hard requirements that MUST be met to meet acceptence criteria.
+ - MUST NOT, SHALL NOT are to be used to establish hard requirements that MUST be avoided in order to meet acceptence criteria.
+ - SHOULD, RECOMMENDED are to be used to establish soft requirements that SHOULD be meet, but deviations are acceptable provided there is a good reason.
+ - SHOULD NOT, NOT RECOMMENDED to be used to establish soft requirements that SHOULD be avoided, but deviations are acceptable provided there is a good reason.
+ - MAY, OPTIONAL are to be used to establish OPTIONAL criteria whose implentation, or lack there of, is left to the developer. This MUST NOT effect core functionality.
+ - The above interpretations SHALL only hold when the given word is fully capitalized
+
+## Git and Version control standards
+- Developers MUST NOT push directly to main. Instead, developers MUST develop feature branches, which are to be merged into main after review.
```

Comment:
Can we add something here for deleting feature branches from the repo once they are merged or closed please?

Reply:
Could have sworn it's already done. Will do